pyproject.toml

```diff
@@ -145,6 +145,7 @@ google-api-python-client = ">=1.7.11"
 pytest-asyncio = "^0.23.5"
 types-sqlalchemy = "^1.4.53.38"
 ruff = "^0.3.2"
+clickhouse-driver = "^0.2.7"
```

Comment:
please create an extra for clickhouse, see how it is done in other destinations e.g. bigquery

Reply:
Fixed in 83b38121b6081afd98d96264a2091a612df559c6